keras_cv/layers/preprocessing/grid_mask.py

```diff
@@ -75,7 +86,7 @@ def __init__(
         ratio="random",
         rotation_factor=0.15,
         fill_mode="constant",
-        fill_value=0,
+        fill_value=0.0,
         seed=None,
```

Comment:
I need `seed` for the `RandomRotation` layer, however `seed` is not a property on `BaseRandomLayer`, so I was forced to access it from the private property `self._random_generator._seed`.

It might be better UX to have it accessible as`self.seed` through parent class, or have random generator be public like `self.random_generator`.